app/models/embed/purl.rb

```diff
@@ -8,7 +8,7 @@ def initialize(druid)
     end
 
     def title
-      @title ||= ng_xml.xpath('//identityMetadata/objectLabel').try(:text)
+      @title ||= ng_xml.xpath('//dc:title','dc'=>'http://purl.org/dc/elements/1.1/').text
```

Comment:
Do we need to guard against the xpath not being there? I'm thinking we should if possible using `try(:text)` rather than getting an error.